tools/xcms_xcmsset/abims_xcms_xcmsSet.xml

```diff
@@ -103,6 +105,7 @@
                 <expand macro="input_centwave"/>
                 <section name="CentWaveAdv" title="Advanced Options" expanded="False">
                     <expand macro="input_centwaveAdv"/>
+					<param argument="peaklist" type="boolean" checked="false" truevalue="TRUE" falsevalue="FALSE" label="Get a list of found chromatographic peaks" help="Generates a tabular file with all the chromatographic peaks obtained." />
```

Comment:
There is an indentation issue here.
Can you use 4 spaces instead of tabulation?